src/components/App.js

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Yes, of course, we could but a lot of folks are using Hooks instead nowadays. I read that using class components is kind of the old way of doing things, and Hooks should be prefered unless we are obliged to use class syntax. The class components vs functional component debate is fierce. I'd have to re-write the whole thing just to meet this requirement and the code would be way more verbose but the functionality would be the same.

Reply:
I totally understand I use Hooks to in my capstone project. I personally prefer functional components to class components. 
I think this could be discussed with a PR or Issue in the project required spec.
But it's the requirement.